config/sources/families/bcm2711.conf

```diff
@@ -166,10 +150,10 @@ function post_family_tweaks_bsp__add_hooks_to_move_kernel_initrd_and_dtb() {
 	run_host_command_logged chmod a+x "${destination}"/etc/initramfs/post-update.d/zzz-update-initramfs
 }
 
-function post_family_tweaks_bsp__add_x11_config() {
+function post_family_tweaks__add_x11_config() {
```

Comment:
I will advice against this change. As the file was added as normal file instead of a conf file, removing it now will break display for anyone already running armbian after upgrade